parser/parser.go

```diff
@@ -26,6 +34,244 @@ func (f File) String() string {
 	return sb.String()
 }
 
+func (f File) CreateRequest() (*api.CreateRequest, error) {
```

Comment:
Possible to add a comment here? It's not totally clear what this is doing, although my assumption is it's creating a new `CreateRequest` provided a `Modelfile`